pkg/cloudprovider/provider/aws/provider.go

```diff
@@ -711,8 +711,19 @@ func (p *provider) Get(machine *v1alpha1.Machine) (instance.Instance, error) {
 		return nil, cloudprovidererrors.ErrInstanceNotFound
 	}
 
+	i := inOut.Reservations[0].Instances[0]
```

Comment:
Can we do do `len` checks before accessing those items?

Reply:
We do already on https://github.com/kubermatic/machine-controller/pull/348/files#diff-4c53f3e4be9b1b46a8d55115391a44eaR710